src/app/SharpRaven/Data/Requester.cs

```diff
@@ -74,8 +75,7 @@ internal Requester(JsonPacket packet, RavenClient ravenClient)
             this.webRequest.Headers.Add("X-Sentry-Auth", PacketBuilder.CreateAuthenticationHeader(ravenClient.CurrentDsn));
             this.webRequest.UserAgent = PacketBuilder.UserAgent;
 
-            if (ravenClient.Compression)
-            {
+            if (ravenClient.Compression) {
```

Comment:
I see you have a different code style than the project. Can you please not change the brace placement? 😃